linux/ibus-keyman/src/meson.build

```
@@ -0,0 +1,43 @@
+utilfiles = files(
+  'keymanutil.c',
+  'keymanutil.h',
+  'kmpdetails.c',
+  'kmpdetails.h',
+)
+
+engine_sources = files(
+  'main.c',
+  'engine.c',
+  'engine.h',
+  'keycodes.h',
+  'keyman-service.c',
+  'keyman-service.h',
+)
```

```suggestion
util_files = files(
  'keymanutil.c',
  'keymanutil.h',
  'kmpdetails.c',
  'kmpdetails.h',
)

engine_files = files(
  'main.c',
  'engine.c',
  'engine.h',
  'keycodes.h',
  'keyman-service.c',
  'keyman-service.h',
)
```

We should use either `_files` or `_sources` consistently.